src/radical/pilot/pilot.py

```diff
@@ -20,9 +20,11 @@ class Pilot(object):
     """
     A Pilot represent a resource overlay on a local or remote resource.
 
+    .. envvar:: RADICAL_TEST_ENV
```

Comment:
I'm not sure what you are trying to demonstrate here. There is a directive, but you aren't using it to provide any explanatory text about the meaning of the environment variable.

Reply:
Sorry I was not supposed to commit this. But I had to demonstrate sphinx behaviour in cases of multiple declarations/definitions of envvar with the same name, and then usage reference behavior. I have removed this.